.github/workflows/codeql.yml

```diff
@@ -0,0 +1,70 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Hi @basbruss,
can you remove this changes?
It creates merge conflicts since I directly changed that file in the main branch.
Greetings